src/pages/Tutorials/Tutorials.vue

```diff
@@ -100,11 +126,30 @@ export default {
 
 .sm-and-up-left-padding {
   // Match left padding with User Settings sidebar width
-  padding-left: 256px;
+  padding-left: 356px;
+  padding-right: 256px;
 }
 
 .sm-and-down-left-padding {
   // Match left padding with collapsed User Settings sidebar width
-  padding-left: 56px;
+  padding-left: 136px;
+  padding-right: 16px;
+}
```

Comment:
Okay I moved the sidebar back to the left and it's now using the `mini-variant` prop but I feel like the padding is still weird when the screen size gets smaller. I feel like there is a better way of doing this but I'm not sure

Reply:
Can you share a picture of what you mean here?  I see it overlap a lot on mobile sized screens but not sure that's what you're referring to?

Reply:
So I think what I meant was when it's on mobile screen sizes and the toc is open, the toc will overlap but not if the toc is closed. I think I was just having a hard time making it look good on mobile screens

<img width="475" alt="closed" src="https://user-images.githubusercontent.com/40726391/99837758-f4bc6c80-2b1c-11eb-8c69-623d32d6743c.png">
<img width="484" alt="open" src="https://user-images.githubusercontent.com/40726391/99837765-f5ed9980-2b1c-11eb-9cf5-9d0750b14b05.png">

Reply:
Just took a look and I think you've got it fixed.  It closed on hover/click for me which meant I could read and navigate fine. Also, I can't see many users wanting to use mobile for the tutorials.